utah-design-system-website/src/react/components/websiteContent/library/components/containers/TabGroup/TabGroupPropsDocumentation.jsx

```diff
@@ -0,0 +1,277 @@
+import {
+  Table, TableBody, TableCell, TableHead, TableHeadCell, TableHeadRow, TableRow, TableWrapper
+} from '@utahdts/utah-design-system';
+
+export function TabGroupPropsDocumentation() {
+  return (
+    <>
+      <h3><code>TabGroup</code></h3>
+      <div className="documentation-content--small-text static-example static-example--blank">
+        <TableWrapper className="static-example__component-wrapper">
+          <Table className="table--lines-x">
+            <TableHead>
+              <TableHeadRow>
+                <TableHeadCell className="text-left">Property</TableHeadCell>
+                <TableHeadCell className="text-left">Type</TableHeadCell>
+                <TableHeadCell className="text-left">Default</TableHeadCell>
+                <TableHeadCell className="text-left">Description</TableHeadCell>
+              </TableHeadRow>
+            </TableHead>
+            <TableBody>
+              <TableRow>
+                <TableCell><code className="primary-color">children</code></TableCell>
+                <TableCell>
+                  <div className="props-code-wrapper">
+                    <code>node</code>
+                  </div>
+                </TableCell>
+                <TableCell><em>required</em></TableCell>
+                <TableCell>
+                  The content of the tab group.<br />
+                  Includes <code>TabGroupTitle</code>, <code>TabList</code>, <code>Tab</code>, <code>TabPanels</code> and <code>TabPanel</code>.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">className</code></TableCell>
+                <TableCell><code>string</code></TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>
+                  The CSS Class to use on the tab group wrapper.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">defaultValue</code></TableCell>
+                <TableCell><code>number</code></TableCell>
```

Comment:
I wonder if defaultValue should now be changed to be a `tabId`, instead of a sequence number, since every tab is now required to have an id and that would be safer in case the tabs get rearranged in the composition. This includes changing the type from `number` to `string`.